### PR TITLE
Don't dublicate event listeners after 'document.open' call in iframe

### PR DESCRIPTION
**Pull request by miherlosev**

### CODE REVIEW

**Comment by miherlosev** [comment]
@LavrovArtem FPR

File: src/client/utils/event-emitter.js [hunk 1]
```diff
@@ -2,51 +2,45 @@ import nativeMethods from '../sandbox/native-methods';
...
+        this.eventsListeners[evt] = nativeMethods.arrayFilter.call(listeners, currentListener => currentListener !== listener);
     }
 
     on (evt, listener) {
-        if (!this.eventsListeners[evt])
-            this.eventsListeners[evt] = [];
+        this.eventsListeners[evt] = this.eventsListeners[evt] || [];
 
-        this.eventsListeners[evt].push(listener);
+        if (this.eventsListeners[evt].indexOf(listener) === -1)
```
Comment (LavrovArtem) [inline src/client/utils/event-emitter.js]: need tests

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
FPR

**Pull request merged**